src/components/ContactsList.vue

```diff
@@ -26,27 +26,65 @@
 			<div class="search-contacts-field">
 				<input v-model="query" type="text" :placeholder="t('contacts', 'Search contacts …')">
 			</div>
+			<Actions
+				class="merge-button"
+				menu-align="right">
+				<template v-if="selected.length >= 1">
+					<ActionButton v-if="showAddToGroup"
+						icon="icon-clone"
+						@click="OpenMultiselect">
+						{{ t('contacts', 'Add to group') }}
+					</ActionButton>
+					<Multiselect v-if="showSelectGroup"
+						:options="groups"
+						:taggable="true"
+						@input="addSelectedContactsToGroup"
+						@tag="addSelectedContactsToGroup"
+						@close="closeMultiselect" />
+				</template>
+				<ActionButton v-if="selected.length >= 2"
+					:close-after-click="true"
+					icon="icon-clone"
+					@click="mergeContact">
+					{{ t('contacts', 'Merge') }}
+				</ActionButton>
+				<ActionButton v-if="selected.length >= 1"
+					:close-after-click="true"
+					icon="icon-delete"
+					@click="deleteMultipleContact">
+					{{ t('contacts', 'Delete') }}
+				</ActionButton>
+			</Actions>
 		</div>
 		<VirtualList ref="scroller"
 			class="contacts-list"
 			data-key="key"
 			:data-sources="filteredList"
 			:data-component="ContactsListItem"
-			:estimate-size="68" />
+			:estimate-size="68"
+			:extra-props={selected}
+			@update-check-selected="selectionChanged" />
 	</AppContentList>
 </template>
 
 <script>
 import AppContentList from '@nextcloud/vue/dist/Components/NcAppContentList'
 import ContactsListItem from './ContactsList/ContactsListItem'
 import VirtualList from 'vue-virtual-scroll-list'
+import Actions from '@nextcloud/vue/dist/Components/Actions'
```

Comment:
@szaimen you have to prefix the component imports

Reply:
hopefully fixed with a9cbbeb